test/multiverse/lib/multiverse/runner.rb

```diff
@@ -95,6 +104,7 @@ def execute_suites(filter, opts)
       "httpclients" => ["curb", "excon", "httpclient"],
       "httpclients_2" => ["typhoeus", "net_http", "httprb"],
       "infinite_tracing" => ["infinite_tracing"],
+      "logger" => ['grpc', 'logger'],
```

Comment:
Why are we creating a new multiverse group?
If this is intended, we should probably rename it since it would be surprising to see grpc in the 'logger' group. Also, we would need to add this group to our CI workflows for multiverse

Reply:
Whoops! That is not intended! It was a group I was using to test the behavior of the reporter. I'll remove!

Reply:
Resolved in: [c9fe5b8](https://github.com/newrelic/newrelic-ruby-agent/pull/1468/commits/c9fe5b86039a7f8ad7ee2a66bfd8df3e413b8dc5)